.gitignore

```diff
@@ -83,11 +83,15 @@ bench-report.xml
 ##############
 
 /enso
+/project-manager
 *.exe
 /enso.exp
 /enso.lib
 *.pdb
 *.jar
+/built-engine/
+/built-project-manager/
+/built-launcher/
```

Comment:
Instead of creating 4 separate directories in the root, I'd create a single one (with subdirectories) to store the build output. IMO this way it makes less noise when you try to `ls` things